src/main/java/com/google/codeu/servlets/MessageServlet.java

```diff
@@ -117,7 +117,16 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     Message message = new Message(user, textWithImagesReplaced, recipient);
 
-    /* This creates a Blobstore instance, then gets the image url(s) which are stored
+    //blobstore function that gets and saves image url
+    setMessageImageUrl(request, message);
+    datastore.storeMessage(message);
+
+    response.sendRedirect("/user-page.html?user=" + user);
+  }
+
+  private void setMessageImageUrl (HttpServletRequest request, Message message) {
```

Comment:
I would add a quick function description of what's happening in setMessageImageUrl